Dockerfile.dapper

```
@@ -62,19 +59,55 @@ RUN mkdir /usr/tmp && \
     mv /usr/tmp/helm /usr/bin/helm_v3 && \
     chmod +x /usr/bin/kustomize
 
-RUN curl -sLf ${!K3S_DOWNLOAD_URL} -o /usr/bin/k3s && \
-    chmod +x /usr/bin/k3s && \
-    mkdir -p /go/src/github.com/rancher/rancher/.kube && \
+# Set up K3s: copy the necessary binaries from the K3s image.
+COPY --from=rancher/k3s:v1.22.3-k3s1 \
```

Would be really nice to get this up to the latest release, since that's the version of K3s that 2.6.3's KDM will recommend.
```suggestion
COPY --from=rancher/k3s:v1.22.4-k3s1 \
```